### PR TITLE
ORM 4 Engies 2016

### PR DESCRIPTION
**Pull request by Alexshreds**
<s> Now engineers can steal all the diamonds</s>
Engineers and ATs are capable of retrieving building materials that they <s>don't</s> need for <s>non</s> engineering purposes.

:cl: Alexshreds
tweak: Engineers and Atmos Techs have access to the ORM
/:cl:

### CODE REVIEW

**Comment by GelatinousGlob** [comment]
Awww yeahhh

**Comment by FalseIncarnate** [comment]
The only real use for this (aside from making diamond doors and such because bling bling) is holding the materials hostage until science finally decides to give me the board and components i've asked for a half dozen times.

**Comment by KingPhilipIII** [comment]
I don't know. I've had busy shifts as engineering where I have to bitch at the miners or QM to give me metal sheets from the ORM.

**Comment by Fox-McCloud** [comment]
CE already has access to the ORM, and mosttt of the time engineers and atmos tech don't need this much metal due to access on the engineer satellite, engineering proper, and EVA--there's a gigantic amount of metal in those areas.

**Comment by Taaketa** [comment]
Only time an engineer needs to be near the orm is for plasma or plasteel. Both of which QM, CE or any miner can supply including cargo techs.

**Comment by Alexshreds** [comment]
I did this not so much because I was big on it, but because people on the forums wanted it.

**Comment by Fox-McCloud** [comment]
>I did this not so much because I was big on it, but because people on the forums wanted it.

Does this mean if I make a forum post to add a sound to *fart you'll make a PR? :smirk:

**Comment by Alexshreds** [comment]
If there is large portion of people supporting it, yes. There is a suggestion thread for a reaaaaason.

**Comment by KorpoPapiez** [comment]
It's just annoying that everytime I wanted some plasma glass as atmos tech I had to explain to bald techs why I couldn't use normal glass and why atmos tech would need it..

**Comment by Sky-Ping** [comment]
...There's... opposition?

Ok, point by point deconstruction, I've gotten good at this, here we go.

> The only real use for this (aside from making diamond doors and such because bling bling) is holding the materials hostage until science finally decides to give me the board and components i've asked for a half dozen times.

The exact same threat comes from Bartenders and Virologists, who both have ORM access.  Plus, the CE can also do this as well.  PLUS, you can deconstruct the ORM to steal all the minerals, which ANYONE that has access to tools (IE, everyone) can do.

Have we ever seen it happen recently?  No.  Has it probably happened before?  Yeah, probably.  Is the threat of it happening enough to prevent engineers access to building materials?  **FUCK** no.

> CE already has access to the ORM, and mosttt of the time engineers and atmos tech don't need this much metal due to access on the engineer satellite, engineering proper, and EVA--there's a gigantic amount of metal and glass in those areas.

Plasteel and Plasma Glass; there is only one stack of plasteel in the engineering lobby, and there is no Plasma Glass in the engineering lobby.  During most major explosions, both plasteel and plasma glass goes missing, meaning you NEED those stacks.  Moving on.

> Only time an engineer needs to be near the orm is for plasma or plasteel. Both of which QM, CE or any miner can supply including cargo techs.

Oh, thanks.  Anyhow: any QM or miner can also just provide the Bartender with Uranium for their drinks, and any QM or miner can provide the Virologist with Plasma for their viruses.  But the Bartenders and Virologists don't.  Because they have access to the ORM.  

And guess what?  Let's say that a PR was made to remove ORM access from the Virologists and Bartenders.  Now the Virologists and Bartenders will simply get a screwdriver and crowbar, deconstruct the ORM, and take the materials.

Can Engineers and Atmos Techs do that?  No, because if the ORM gets deconstructed, it will not drop Plasteel or Plasma Glass; it needs to be smelted inside the ORM.

In conclusion: there is still absolutely no reason why jobs that are vital to the maintenance of the station should be denied access to materials, when jobs that are purely for pleasure are allowed access to materials.  The sheer thought of this PR gaining any type of resistance whatsoever completely blows my mind.

**Comment by Kyep** [comment]
Ping's arguments are convincing. +1

If bartenders have ore machine access, so should engineers.

**Comment by Davdichan** [comment]
Miners, QM, CTs, HoP, Cap, RD, Science, Robotics, CE, Mechanic, Chemist, Virologist and Bartender all have access to the machine that I know of. The fact that Engineers are specifically being excluded here is ridiculous, either anyone with a need for the materials as part of their job should get access, or it should be restricted to Cargo only.

Bout the only thing engineers want is metal, glass and plasteel and plasglass anyways, which are the three most common 'ores' mining can get. So I don't see any reason to restrict them, if they are found to be stealing bling for autism forts than the miners or science teams might just remodel with some boom.
